### ISSUE TITLE
Test 2

### ISSUE THREAD

**Issue by josephdelon** [DrasilRNA/test#51]

**Comment by jilldevont** [comment]
I know that the giant red thing above the dooris is the switch but i dont know how to reach it!!!

**Comment by jevoneorwells** [comment]
I am in wolf form. I did not get the Lake Hylia teleport. Everything is frozen over, So using rivers for transportation is out of the question. I saw on these forums that I should go to the grass near the howling stone. But.. I cannot get to that half of Upper Zora's River. I haven't talked to the spirit. Should I reset the game file?

**Comment by jevoneorwells** [comment]
.nosaer on rof gniyonna tsuj fo adnik si emag siht tniop siht ta ,sreliops htiw yako %001 ma I ?ruof lanif eseht teg I od woH .elpmaxe rof revir eht no gnihtemos ro egdirb a morf egakcolb nekorb a si ereht ;emag ni dekcolb yllaicifitra era setuor eht fo lla dna aera taht ot og ot gniyrt peek I .edis revir thgir eht no sgub thgil ruof lanif eht teg tonnac em fo efil eht rof tub ,niamod s'naroZ ni aera thgiliwt lanif eht no ma I

**Comment by jevoneorwells** [comment]
?snoitseggus ynA .emit ni ereht gnilbmob eht teg t'nac I tub kcor a pu wolb ot evah I dna ecalp elpmet eht ni yeknom htruof eht teg ot gniyrt m'I kO

**Comment by jevoneorwells** [comment]
Fortune teller says my last heart piece is at ordon ranch but its locked and village is empty. I could have sworn I got this one but in her love vision there is no mistaking that I am seeing the ranch with all the wood plates around it flapping in the wind. I cant get my horse to jump it and I cant dig in. What am I missing? Is it possible he gave me the heart piece and I didn't grab it? Am I doomed to start over? I have 109hrs logged into this game and I've already started work in the castle dungeon so that would really suck. Before I learned of the gypsy I had already retraced my steps through half the game so I'm not really looking forward to starting from scratch.Thanks in advance.-Gus??

**Comment by jevoneorwells** [comment]
Ok, I'm in the forest temple and I just beat the big plant thing and rescued the third monkey. I went to the main room again, and decided to go to a diffrent room. The monkey fell off the main platform thing, and I went into the other room with only two monkeys. Surprised, I went back into the main room to see the monkey that fell down now dancing on top of a pillar. I have no idea why he is doing this, but I know I need him in order to progress the temple. Any suggestions? It didn't say a thing about this in the guide I was following..

**Comment by jevoneorwells** [comment]
?sgorf hctac uoy od woH

**Comment by jennywaiver** [comment]
I can't beat Roll Goal for the frog bait so I need someone to tell me how to cacth it without it. I have looked at the Fishing Faq but it didn't help much.

**Comment by johnasduval** [comment]
How doIbeat the giant knight in the mansion? I've tried just about everything and nothing works.and can anyone tell me how to use the game saves?

**Comment by jennywaiver** [comment]
?sdriB ro srehcrA tsrif tuo ekat I od ohw oS .wonk t'nod tsuj I ,sdrib eht rof sa dna tuo ekat I ynam woh rettam on erom neve nwaps ot mees srehcra dekcabdeggoh eht taht smees ti tub ,pu emoc yeht sa sdrib eht teg ot yrt emit emas eht ta dna ,sworra ym htiwkcabgoh no srehcra eht tuo ekat ot yrt I ,elbuort evah I taht etag eht kcolnu I retfa s'tI .etag tsrif eht kcolnu uoy erofeb yako trap tsrif ehthguorht teg I ,nogaw a trocse uoy erehw ,noissim trocse eht no m'I

**Comment by johnasduval** [comment]
How do i find the third suit? I have found the Zora armour and i have defeated Gandorf, but dont know where to get the third suit.

**Comment by jevoneorwells** [comment]
How do I get past the two bridges at the forest temple

**Comment by jennywaiver** [comment]
dnammoc eht tresnier.evorg dercas ot nruter nI

**Comment by jevoneorwells** [comment]
I have gotten the statue all the way down to the main floor and put it into place. I cannot move it now. I need it to get the boss key that I didn't know how to get earlier. There are only three statues in the room and no way for me to get any more up there. I must use the statue on the fourth spot. Is there any way that I can do this without using Ooccoo? Thanks.

**Comment by jennywaiver** [comment]
I would like to know if the mirror shield is an option to beating the cave of ordeals 5 times in twilight princess?

**Comment by johnasduval** [comment]
PLEH ESAELP.ssecnirp thgiliwt ni nilbog gib eht taefed ot tfel sbmob yna evah t nod I dna dik aroz eht pleh ot gniyrt kcuts ma I

**Comment by jennywaiver** [comment]
.emag eht ni reven saw tI ?ereht ti si yhW ?nihtemos ro rab cigam eht ti SI .moc.adleZ etisbew eht no scip eht htiw emas ehT .traeh eht rednu RAB NEERG a htiw serutcip sereht ,revoc emag eht fo kcab eht no ,yeH

**Comment by jephrynorman** [comment]
.em deppots yeht dna eloh gnihsif eht ot teg ot gniyrt ma I .flow a otni kcab degnahc I nehw retal meht fo erac koot tsuj dna )egdirb eht no( deppiks I mrof namuh ni od ot dah I eno tsal eht

**Comment by johnasduval** [comment]
I accidentaly said no to the skeleton when he told me if I wish to learn the next hidden skill.Now the golden wolf is gone and the howling stone is inactive. What do I do to fix this? i already had saved as i thought the stone would work again Please help

**Comment by johnasduval** [comment]
?rettal eht fo pot eht ta nogard eci eht taeb t'nac i tub ,tsaf ecalp eci siht worht teg ot tnaw I

**Comment by jephrynorman** [comment]
Hey, I can't get the last hidden skill in Twilight Princess HD. I'm 100% sure I howled at all the stones because every single one has already been activated and you obviously can't use one twice. I need the Great Spin technique. And there isn't a Wolf anywhere on my map. I think my game is broken...Can someone help?

**Comment by jevoneorwells** [comment]
.pleh esaelp ereht sgniht doog emos era ereht yltnerappa dna si ti erehw wonk t'nod i tub saroz eht raen eloh gnihsif doog yllaer a si ereht taht syas enoyrevE

**Comment by jevoneorwells** [comment]
The last Hidden Skill I needed was the one from the Hidden Village. I howled at the Howling Stone and the Golden Wolf appeared on my map, as per usual. I went and interacted with him, but when the creature who teaches you (I dont know his name) asked me if I was ready to learn, I accidentally said "no". I can no longer get the Golden Wolf from the Howling Stone. Is there any way I could possibly still learn the Hidden Skill?

**Comment by jephrynorman** [comment]
I am stuck here. It's really hard to put the fire out, I can't lock the Gale Boomerang properly, it often locks on the ground or something. That's the main problem. Also, is there a way to defeat the Orcs? They always come back.Please try to help me! I love this game, but I'm stuck.

**Comment by johnasduval** [comment]
?kcab reh teg I nac woh dna ,og ehs did erehW !enog s'anopE dna ,sniatnuom eht ot ot og ot desoppus m'I won erehw ,thgiliwt fo dnal eht fo trap dnoces eht tuo gniraelc dehsinif tsuj I

**Comment by jennywaiver** [comment]
/ti fo tuo teg u od taw dna ,yalp u od woh ,ti si erehw ,laogllor si taw ,pam laoglloR a saw eno yeht dna sediug emos gnidaer saw I

**Comment by jephrynorman** [comment]
I know that other people think about this, so what's up with Impaz's name? Is she a relative / sibling of Impa? Or is she just a knock-off?

**Comment by jevoneorwells** [comment]
I can`t get past the top east side of the water temple i have got past the bottom but not the top please help me

**Comment by jevoneorwells** [comment]
.dleiF eluryH ta goh gib eht no ssob eht s'tI .ecnivorP nidlE eht ta s'tI

**Comment by jevoneorwells** [comment]
I have just gone through goron mines and i am back in the villiage. The kid sells the hylian shield for 200 ruppees but after getting the hawks eyes and the bomb bag i dont have enough to buy it. I want to get it before i go to the next twilight area in hyrule. Any tips of how to get lots of ruppees?

**Comment by jephrynorman** [comment]
I've found two but I can't find the third? plz help

**Comment by jevoneorwells** [comment]
REDAV-LAS ?gnorw gniod I ma tahw rO ?ni teg I od woh os fi dnA ?uryanaL fo tiripS eht fo evac eht siht sI .eci ot eud elbartenepmi si dnif I evac ylno eht ,eluryH ekaL ot yaw ym no llafretaw eht nwod gniog dna aletuR htiw gnikaeps retfAniamoD aroZ :eR

**Comment by jennywaiver** [comment]
???senim norog eht ni retal dleihs eht sI

**Comment by jennywaiver** [comment]
?emit emas eht ta sbmob retaw dna sgnilbmob yrrac nac I yaw a ereht sI

**Comment by jephrynorman** [comment]
.u iiW eht no si sihT .t'nac tub did I ,kcits yoj thgir eht esu ni mooz ot syas ehs dna kcahs gnihsif eht ta m'I

**Comment by jennywaiver** [comment]
egaliv eht ot yob noroz eht gnidael era uoy nehW

**Comment by jephrynorman** [comment]
!!PLEH .gniht llaw citats eht fo edistuo si meht fo eno tub owt rehto eht llik ot ydaer teg i neht eno llik i tsrif

**Comment by jennywaiver** [comment]
!tirips thgil eht ot deklat ev'I litnu emag eht htiw deecorp t'nac I sa em pleh esaelP ?ailyh ekal ot teg ot yaw rehto yna ereht sI .kcuts yllacisab m'I .ffo pmuj ot esu ot em rof xob on si ereht sa niaga egdirb eht ffo pmuj t'nac I .ereht teg t'nac I dna tsrif niamod saroz ot gniog tuohtiw ailyh ekal ot teg t'nac I tub ,emag eht htiw deecorp ot ailyh ekal ot og ot deen I .hcaer t'nac I hcihw niamod saroz ni eno eht si si ereht eno rehto ylno eht dna ,eno esu ot thgil fo dlrow eht ot kcab em tel t'now andim sa gniht praw a esu t'nac I ?nosaer emos rof redluob a yb dekcolb won si niamod saroz ot kcab yaw ylno eht dna ,fo tuo emac I taht lennut revir eht hguorht pu kcab miws em tel t'now ti sa niamod saroz ot kcab og t'nac I .ereht morf ailyh ekal ot teg ot woh wonk t'nod I .dleif eluryh ni revir eht ni pu dedne dna tnerruc gnorw eht nwod tnew yltnedicca I tuB .nezorfnu si ekal eht won ailyh ekal ta tirips thgil eht ot klat dna og ot em dlot andiM .niamod saroz ta eci eht morf saroz eht deerf tsuj ev'I dna mrof flow ni m'I os ,yakO

**Comment by jephrynorman** [comment]
)gnitairufni si edistuo morf moor emas eht ot kcab gnitteg tub ,devlos niamer selzzup wonk I( ?ssergorp ym gnisol tuohtiw evas I od woH ?nogenud eht edisni evas tsuj I oD.nogenud eht otni kcab praw ot noitpo on htiw ,ooccoO yb decalper si dna enog si .rJ ooccoO tub )pohs eht ot esolc eno eht( !retaw eht fo elddim eht ni kniL snwaps emag eht ,evas ym dedaoler I nehw tub )noitpo ylno eht s'taht ,pohs eht ot( "edistuo" praw ot ooccoO desu I .krow ot mees t'nseod siht ykS eht ni ytiC no ,tuB .tfel I taht moor emas eht ot yltcaxe ,edisni kcab praw ot .rJ ooccoO esu neht ,gniyalp pots ot tnaw I nehw evas dna edistuo praw ot ooccoO esu I nogenud ni ,yllausU

**Comment by jevoneorwells** [comment]
?ylisae ti teg ot hsif I dluohs erehW .erul gniknis eht dnif ot gniyrt mi dna ,erul gorf eht tog osla I .gnihsif erul hsiftac nodro eht dna ,ekip ,ssab nailyh eht thguac I

**Comment by jevoneorwells** [comment]
!pleH !htiw mih toohs ot sworra fo tuo m I tub nwapser I naht gniyd peek tub semit fo hcnub a mih tog I

**Comment by jennywaiver** [comment]
mih taeb i od woh frodnonnag fo mrof gip eht elytab u erehw kcuts mi kO

**Comment by jennywaiver** [comment]
Ok i just got through this door in the chandelire room and i lit two torches and blew out the lit on and got up o the chest i went to the left and killed all the bats in that room and lit all the torches but one kept going out and the door remaind shut i then went to the right room and killed the two monsters in that room the door remained shut i am now stuck please help.

**Comment by johnasduval** [comment]
I was almost to the end of the Sky Temple and then I missed the ceiling switch to turn on the fan, I figured since I saved after getting the double clawshot I could just reload the game. WELL, now I have NO double clawshot and the bridge was already broken by the dragon. So what the heck can I do so I don't have to start the game over again because I'm livid!

**Comment by jephrynorman** [comment]
Is it between majara's mask and windwaker? if not when?

**Comment by johnasduval** [comment]
I have killed all the insects for the tear of light in Faron Woods. I killed both of the insects in Coro's House but apparently was only able to collect one tear - the map still shows that a tear is inside, but even when I press "C" to look around the room, I cannot see it. I reset the game and no change. Do I really have to start the whole game over, or is there a way to collect this tear? I can't target anything in the room. Tried charging in to the walls to see if I could knock something down, no luck.

**Comment by jevoneorwells** [comment]
In the room to the the right of the first room, there's a chest across a pit. How do I get over there?

**Comment by jevoneorwells** [comment]
I read somewhere that if you clear the cave of ordeals 5 times you will be given the mirror shield. Apparently it only is helpful in the final fight to not have to swing your sword but...I did it a second time you was just not up to doing it 3 more times without someone telling me for certain it is true. I mean I like to use the fairy tears once so I don't have 2 of them and can't get them without doing it:(.

**Comment by johnasduval** [comment]
ereht si lleW

**Comment by jevoneorwells** [comment]
I tried to get a fairy from eldin springs and i missed and i got plain water so what is that good for?

**Comment by jevoneorwells** [comment]
.seod ti kceh eht eht wonk ot tnaw I .gniyona yllaer sti .yawa snur ti tac eht hcaorpa uoy ecnO .tac dnoces a si foor eht no ,elttob klim eht niatbo uoy dna ,hsif eht tac eht evig uoy retfA ?gninigeb eth ni foor pohs eht pot no yttiK eht htiw od ot tahw wonk enoynA

**Comment by jephrynorman** [comment]
!pleh esaelP ?hctilg a ti si ro eno ssim i diD ?taht syas ti yhw nialpxe s'ydobemos naC .ecnivorp hcae rof lla meht evah i syas pam ym tub inavoj ot klat i nehw dna neercs noitcelloc ym ot gnidrocca eop eno gnissim ma I

**Comment by jennywaiver** [comment]
?elif evas ruoy tceles uoy nehw deyalp cisum eht fo eman eht em llet enoemos dluoC

**Comment by jevoneorwells** [comment]
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!PU STAHW eceip traeh ym teg reven I tub seepur 05 etanod syawla I !!!eceip traeh ym teg reven I tub yenom etanod uoy sekam taht nam eht morf eht eceip traeh a gnidnif pleh deen I

**Comment by jephrynorman** [comment]
?pleh....ti kcolnu ot yek eht evah t'nod i haey dna saw rood dekcol eht erehw pot eht ot tog dna sliar llaw eht pu tnew dna rennips eht gnisu moor eht dekcolnu dna roolf eht no sraeg eht htiw moor eht ot tnew i dna )kniht i pu dewercs i erehw staht( emag eht detratser i oS .kcuts atros tog i neht dna dnuora teg ot rennips eht gnisu hguorht og uoy taht moor eht ot deunitnoc neht dna rennips eht tog Itib a pu dewercs atros I

**Comment by jennywaiver** [comment]
!!!DEWOLLA SBOON oN ,ylnO snareteV adleZ .ssecnirp thgiliwT adleZ netaeb evah u fi ylper tsuJ tuB ,noitseuQ a evah tnoD i LOL

**Comment by johnasduval** [comment]
!srewsna deen i !pleh !tuoba gniklat s'ehs tahw aedi on evah i tub stsalb erif eht htiw ti emit ot em dlot andiM !deneppah gnihton tub yaw eht lla ti dellup i dna senim norog eht ta llaw eht sevom taht ygniht a llup ot evah I

**Comment by jevoneorwells** [comment]
With the Dominion Rod, what do I do to the owls that I find on the map?

**Comment by jennywaiver** [comment]
Wanting to get the full experiance of the game I love finishing all the little quests. I just want to know how many hearts I can have total to be sure I'm getting the full feel, and how many bottles can you have?

**Comment by johnasduval** [comment]
hacerT,sknahT.ti etaicerppa yllaer dluow I evig nac syug uoy ecivda/pleh ynA?neddus a fo lla hctilg a eb ti dluoC .tahw ro gnittegrof tsuj m'I dna thguac eb nac ti erofeb tsrif gnihtemos od yllautca ot evah I fi rebmemer t'nac I .ereht ton llits si ti dna yad eht gnirud dna thgin ta deirt evah I .dnuof eb ot erehw on si ti rood tnorf eht yb eutats romra eht no niahc dna llab ym esu I nehw nosaer emos rof revewoh emit sihT .meht gnitteg melborp a dah reven dna erofeb sluos eoP eht lla nettog ev'I .ti gniyalp-er m'I won dna eciwt emag siht taeb ydaerla evah I ,thgirlA.tuo neeb sah emag siht ecnis emit gnol a neeb sah ti wonk I .sregnif ym gnissorc...siht htiw pleh elttil a rof gnipoh m'I

**Comment by jennywaiver** [comment]
.ylf ot hcihw htiw snekcihc on llits ;cinut aroZ eht nettog dna okirakaK ot silaR thguorb tsuj ev'I ?raeppa soccuc esoht od tneve hcihw retfa ,si wonk annaw I llA.tuoba gninnur s'occuc yna t'nera ereht ,dloheb dna wol tub ,ti retfa m'I os ,tnemom eht ta yenom no thgit m'I si eussi yM .htiw dnuora ylf ot occuc a gnisu ereht pu teg ot elbissop osla s'ti tuB .mrof flow ni hcnic a si ereht pu gnitteg os smrof neewteb hctiws yleerf ot elba gnieb dna drowS retsaM eht gniniag litnu tiaw ot lamron smees tI .ereht pu gnitteg si eussi ylno eht neht tub ,esuoh s'odaneR no rewot lleb eht ta worra bmob a gnitoohs yb eriuqca nac uoy taht egalliv okirakak ni eepur revlis a s'ereht taht erawa llew m'I

**Comment by johnasduval** [comment]
I just got the Double Clawshot from the

**Comment by jennywaiver** [comment]
I've been Madame Fanadi's and this guy pops up when I do the love fortune. I've donated over 300 rupees to this guy and all does is wish me love. Dude, if you've got a heart piece, then cough it up!!

**Comment by johnasduval** [comment]
At Death Mountain, there was a poe soul while you lauch yourself south west at the second goron. I killed it as wolf. It just turned morning. It shoed me getting the soul, but it didnt say anything. Next time, I killed it, cause it was there. Did I get an extra poe? What happened?

**Comment by jennywaiver** [comment]
so, i've finished the game, and saved at the end, but when i load the finished save file, i end up in the room before the final boss. (i haven't played this game in a while, so sorry if it's vague) i thought it might be because you need to delete the file first, but i don't want to take the risk. Is there anything else i have to do before beating the final boss to keep link's starter clothes throughout the whole game?

**Comment by johnasduval** [comment]
At the Warp zone Where you had to fix the bridge in Hyrule field. ( the one where there are always pests on beasts shooting at you) On the bride facing the entry to said field, there's a ladder to the left where Midna says, " there's something over there but there isn't a ledge to jump to..." or something close to that. I've tried blowing things up for rocks to fall to make one, I went around the whole building and nothing. Even tried the clawshot... still nothing... How do I get over there?

**Comment by johnasduval** [comment]
!pleh zlp .sruo rof taht gniod neeb evI .gnihsif rebbob og dna dluoc I sa tsae raf sa og oy dias eht dna tenalptaehc dna srenniwemag ot tnew I nac i dnihtyreve deirt evI

**Comment by jennywaiver** [comment]
?ecar ateY taeb ot skcirt ro spit ynA

**Comment by johnasduval** [comment]
I can't defeat those, I usually hold B and then release it, they all die, I can't move, and they are revived.
Please I need help.

**Comment by jennywaiver** [comment]
I never bought a wii or wii u. I do own a switch and have played through Botw multiple times. But I definitely felt like I missed out on a few zelda games like skyward sword and twilight princess (I know it was a gamecube game. I never got that game) wii u is now $150 at gamestop. Would it be worth it to get a wii u for those two zelda games? And are there any other games that would be worth getting a wii u for? Thanks for the help

**Comment by johnasduval** [comment]
How do I get passed the gorgon protector? I've tried my brother has tried but how do we do it????

**Comment by jennywaiver** [comment]
I need to know so i can skills? so can someone please help?

**Comment by johnasduval** [comment]
Is there an easier way to beat him? I have tried 5 times and don't want to use up all my bottles of life (as I need them at the end boss to beat) and he still gets to knock me off and I only hit him once or twice and Zelda sure can't shot often.

**Comment by jennywaiver** [comment]
Wen im in the graveyard and isense by the tree and the ghosts are pointing at the rock in front of the tree, wat do i do with it?

**Comment by jennywaiver** [comment]
Is there a possibility that I never inside a room in a dungeon,for example,I feel like I missed a couple chests or rooms in Forest Temple but I can't go back much since the monkeys have left and I can't even use the clawshot as it doesn't work on ropes. Can there be secret rooms you never went to inside a dungeon? I have just finished the Temple of Time,if anyone needs to know.

**Comment by johnasduval** [comment]
!!!pleeeeH .ti teg uoy woh dna si ti tahw gnirednow saw I dna ,unem ym no kniL rof tols tiftuo erom no serehT

**Comment by jennywaiver** [comment]
?nwoT eltsaC fo edis htuos eht morf lennut eht gid nac eh os retaw eht noroG eht evig I od woH

**Comment by johnasduval** [comment]
So I saved and quit during the carriage escort and when I logged back in, it put me in Lake Hylia on some land down there. I made it back to the bridge where you re supposed to fight king bobo or whatever and he s not there even tho it shows it on the map. Another issue is that it shows that the carriage is in Lake Hylia on some land too and it s also not there. I can t figure out what to do or how to fix it. Anything? Btw this is on Wii.
